packages/ag-charts-community/src/chart/toolbar/toolbar.ts

```diff
@@ -222,6 +226,21 @@ export class Toolbar extends BaseModuleInstance implements ModuleInstance {
         this.translateFloatingElements(FloatingTop, false);
     }
 
+    // AG-12695 Temporarily set `pointer-events: none` on the annotationOptions when dragging, because the
+    // buttons block to mouse from hovering over the canvas.
+    private readonly oldPointerEvents: Map<HTMLButtonElement, string> = new Map();
+    private onStopPointerEvents() {
+        if (this.oldPointerEvents.size !== 0) throw new Error('AG Charts - unexpected drag-start event');
```

Comment:
I'd avoid a hard fail here, it could leave the chart in an odd/unrecoverable state?

Reply:
I believe that the error logging is useful because this should never happen in theory.

I've replaced it with `Logger.errorOnce` to avoid unrecoverable states.